examples/idgnn_link.py

```diff
@@ -110,7 +110,7 @@ def train() -> Dict[str, float]:
     total_steps = min(len(loader_dict["train"]), args.max_steps_per_epoch)
     for batch in tqdm(loader_dict["train"], total=total_steps):
         batch = batch.to(device)
-        out = model.forward_id_awareness(
+        out = model.forward_rhs_readout(
```

Comment:
@rusty1s Just realized that "rhs" is not common lol

Reply:
`dst_readout`? :)